pages/Developer/MAP API/map-migration-api.md

```diff
@@ -0,0 +1,164 @@
+# Map Migration API Documentation
+
+## Introduction
+
+The Map Migration API provides endpoints to manage and retrieve information about map migration projects. This includes listing all projects, retrieving details of a specific project, listing products within a project, and listing resources within a project.
+
+Try it in our [Public Swagger](https://app.nops.io/public_swagger/)
+
+## Endpoints
+
+### 1. List All Map Migration Projects
+
+- **URL**: `/api/map-migration/`
+- **Method**: `GET`
+- **Description**: Lists all map migration projects for the current client.
+
+#### Request Parameters
+
+None.
+
+#### Response Format
+
+The response is a JSON array containing a list of map migration projects. Each project has the following fields:
+
+- `id`: The ID of the map migration project.
+- `on_prem_non_commercial_db_entity`: Specific incentives related to on-premises non-commercial database.
+- `on_prem_commercial_db_entity`: Specific incentives related to on-premises commercial database.
+- `ec2_non_commercial_db_entity`: Specific incentives related to EC2 non-commercial database.
+- `ec2_commercial_db_entity`: Specific incentives related to EC2 commercial database.
+- `sap_entity`: Specific incentives related to SAP.
+- `baseline_s3_entity`: Specific incentives related to S3.
+- `speciality_services_entity`: Specific incentives related to speciality services.
+- `private_equity_entity`: Specific incentives related toprivate equity.
+- `tagged_resources`: Information about tagged resources, including resource count and cost.
+- `untagged_resources`: Information about untagged resources, including resource count and cost.
+- `credits`: Information about received credits, including records of credit details.
+- `created`: The creation date of the project.
+- `modified`: The last update date of the project.
+- `name`: The name of the project.
```

Comment:
The name of the MAP project. This is also something you specify while onboarding the MAP project in nOps MAP Dashboard and it can be renamed later as well.